shared/constants/oracle.ts

```diff
@@ -0,0 +1,38 @@
+export const currencies = ["USDC", "ETH", "RY"] as const;
```

Comment:
defining these values as constant lists and deriving types from them lets us use these lists to generate exhaustive mocks by iterating over them.

Reply:
i think since this is in shared, we shoudl assume these coudl be real values used in prod, eitehr that or label the file as storybook specific somehow

Reply:
Yeah sorry I think I was unclear. I meant that these are supposed to be the source of truth of the values used in production. then we can use these same values to generate exhaustive mocks of all the possible values we might see in production and make sure that they work in our stories